### ISSUE TITLE
Disable Stress Realism when replaying

### ISSUE THREAD

**Issue by rmroc451** [nguyenquyhy/Flight-Recorder#62]
I was scrubbing through the history of a flight recently, and had my flight fail back to main menu because I overstressed the aircraft.

I was paused in the recorder, and dragging the timeline forward quite fast when it happened.

Might be a nice feature for those of us that don't want to remember to toggle realism on and off?

**Comment by nguyenquyhy** [comment]
How to do that though? 😂

**Comment by rmroc451** [comment]
> How to do that though? 😂

Yeah, I know, maybe when its available to be interacted with through the api... slap enhancement on it and a simconnect api label and call'er good? 🤣
